types/test.ts

```diff
@@ -0,0 +1,9 @@
+import remark = require('remark')
+import remarkAutolinkHeading = require('remark-autolink-heading')
+import hastscript = require('hastscript')
+
+remark().use(remarkAutolinkHeading)
+remark().use(remarkAutolinkHeading, { behavior: 'wrap' })
+remark().use(remarkAutolinkHeading, { group: hastscript.h('element') })
+remark().use(remarkAutolinkHeading, { content: hastscript.h('element') })
```

Comment:
Generally looks good, it may make sesne to use an HTML element with hastscript for clarity

```suggestion
remark().use(remarkAutolinkHeading, { group: hastscript.h('div', 'example') })
remark().use(remarkAutolinkHeading, { content: hastscript.h('div', 'example') })
```